### PR TITLE
show read result.

### PR DESCRIPTION
**Pull request by Kai-Kuze**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -26,3 +26,5 @@ log = "0.4"
 uuid = { version = "0.8", features = ["serde", "v4"] }
 
 firestore-db-and-auth = "0.6.1"
+
+once_cell = "1.3.1"
```
Comment (asana17) [inline Cargo.toml]: I think this dependency is not needed.

File: src/contents/clothing/read.rs [hunk 2]
```diff
@@ -7,6 +7,8 @@ use crate::{
 };
 use firestore_db_and_auth::documents;
 
+use once_cell::sync::Lazy;
```
Comment (asana17) [inline src/contents/clothing/read.rs]: Please remove these lines too.

**Review by asana17 — COMMENTED** [review]
The css header looks so large. Please fix this.

**Review by asana17 — COMMENTED** [review]
Please remove warnings when building.

File: src/contents/clothing/read.rs [hunk 3]
```diff
@@ -100,3 +123,20 @@ struct DTOClothing {
...
+        }
+    
+    }
+}
+#[derive(Serialize, Deserialize)]
+pub struct DTOClothing_string{
+    brand: String,
+    year: String,
+    month: String,
+}
```
Comment (asana17) [inline src/contents/clothing/read.rs]: I think this struct is not needed.

File: src/contents/clothing/read.rs (hunk 2 above)
Comment (Kai-Kuze) [reply src/contents/clothing/read.rs]: i did.

File: Cargo.toml (hunk 1 above)
Comment (Kai-Kuze) [reply Cargo.toml]: i did.

File: src/contents/clothing/read.rs (hunk 3 above)
Comment (Kai-Kuze) [reply src/contents/clothing/read.rs]: i did.

**Comment by asana17** [comment]
Thank you for the commits, but we first need to follow the current main branch changes. I think the implementation for the read_list function is already deprecated.

**Comment by asana17** [comment]
I checked this version, but the functions not working as we wanted them to be.

1. With the read function with the specific document ID, we can see only the brand.
2. With the read_list function, we can't see the document ID correspond with the object in the list.
3. The CSS header still looks so large.

**Pull request closed without merging**